ciongke/cmd/hook/main.go

```diff
@@ -47,6 +47,26 @@ var (
 var defaultJenkinsJobs = map[string][]JenkinsJob{
 	"kubernetes/kubernetes": {
 		{
+			Name:      "kubernetes-pull-test-unit-integration",
+			Trigger:   regexp.MustCompile(`@k8s-bot (unit )?test this`),
+			AlwaysRun: true,
+			Context:   "Jenkins unit/integration",
+		},
+		{
+			Name:      "kubernetes-pull-verify-all",
+			Trigger:   regexp.MustCompile(`@k8s-bot (verify )?test this`),
+			AlwaysRun: true,
+			Context:   "Jenkins verification",
+		},
+		{
+			Name:             "kubernetes-pull-build-test-e2e-gce",
+			Trigger:          regexp.MustCompile(`@k8s-bot (gce )?(e2e )?test this`),
+			AlwaysRun:        true,
+			Context:          "Jenkins GCE e2e",
+			CommentOnFailure: true,
+			RerunCommand:     "@k8s-bot gce e2e test this",
```

Comment:
Because it's the only one that comments when it fails, so it needs to tell users what to say to rerun it. I have a test that the rerun commands actually match the job that they're supposed to.